<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostQuickDialog</class>
 <widget class="QDialog" name="HostQuickDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>接続先</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>ホスト名(アドレス)またはURL(&amp;H):</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_Host</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Host">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ユーザー名(&amp;U):</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_Username</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>パスワード/パスフレーズ(&amp;P):</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_Password</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_Username">
       <property name="maxLength">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Password">
       <property name="maxLength">
        <number>80</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_Firewall">
       <property name="text">
        <string>FireWallを使う(&amp;F)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox_PASV">
       <property name="text">
        <string>PASVモードを使う(&amp;V)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HostQuickDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HostQuickDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
